lib/net/dns/packet.rb

```diff
@@ -520,9 +520,16 @@ def new_from_data(data, from = nil)
         
         @answer = []
         @header.anCount.times do
-          rrobj,offset = Net::DNS::RR.parse_packet(data,offset)
-          @answer << rrobj
-          @logger.debug rrobj.inspect
+          if (rrobj, new_offset = Net::DNS::RR.parse_packet(data, offset))
```

Comment:
This might be a dumb question on my part; In the future would we want to add support for `AFSDB` records to the net/dns implementation that we have, or contribute that upstream?

I've never seen this record before, but based on the description it seems like might be useful information to a pen tester?

> AFSDB records are used for locating AFS and OSF type database resources through the DNS

Reply:
I would think add it. We're currently on 0.4 while upstream is on 0.9 so we're pretty far behind though. I did a quick check and it still didn't look like the newer version supported that RR type but I didn't test if it skipped over unsupported types like this change does.

Reply:
Awesome, not a blocker for me of course. The world is a better place now :+1: